### PR TITLE
Fix BlockfrostSpec on Windows

### PR DESCRIPTION
**Pull request by Unisay**
- [x] I have used relative path to socket counting on it being correct path on all OSs.

### CODE REVIEW

File: lib/shelley/test/unit/Cardano/Wallet/Shelley/Launch/BlockfrostSpec.hs [hunk 1]
```diff
@@ -38,10 +37,8 @@ import UnliftIO.IO
 spec :: Spec
 spec = describe "Blockfrost CLI options" $ do
     it "modeOption --node-socket" $ do
-        -- Named pipes are used on windows
-        pendingWith "Fails on windows: https://github.com/input-output-hk/cardano-wallet/pull/3160#issuecomment-1071163285"
         let parserInfo = info modeOption fullDesc
-            args = ["--node-socket", "/tmp/file"]
+            args = ["--node-socket", "test.sock"]
```
Comment (Anviking) [inline lib/shelley/test/unit/Cardano/Wallet/Shelley/Launch/BlockfrostSpec.hs]: I suspect you'll need something like `"\\\\.\\pipe\\test"` (escaped form) in the case of `ifWindows == True` to make CI green.

https://docs.microsoft.com/en-us/windows/win32/ipc/pipe-names

**Review by Anviking — APPROVED** [review]
lgtm provided it passes!

**Comment by Unisay** [comment]
bors merge

**Comment by Unisay** [comment]
bors merge

**Pull request merged**